.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

Comment:
It's best to do commit this, as it pins the versions of dependencies (also transitively).